styles/globals.css

```diff
@@ -11,18 +11,25 @@ html {
   --dark-background-color: #20272f;
   --light-text-color: #ffffff;
   --dark-text-color: #20272f;
+  --dark-new-color: #022b3a;
 
   /* font families */
   --sans-serif-font: "Roboto";
   --serif-font: "IBM Plex Mono";
 }
 
+.top
+{
+  background: rgb(255,107,107);
+  background: linear-gradient(160deg, rgba(255,107,107,1) 4%, rgba(215,97,99,1) 13%, rgba(96,67,76,1) 44%, rgba(2,43,58,1) 80%);
+}
+
 .page {
   height: 100%;
   margin: 0;
   padding: 0;
   /* background-color: var(--neutral-color); */
-  background-color: var(--dark-background-color);
+  background-color: var(--dark-new-color);  
```

Comment:
use the existing `--dark-background-color` here (but change the value to the new color)